tests/processes/ProcessingTest.py

```diff
@@ -234,6 +241,7 @@ def test_dead_processes(self):
             ctrlq, {}, {}, {},
             lambda *args: self.queue.put(args[2]),
             Section(""),
+            None,
```

Comment:
can you make an integration test in coalaTest that runs coala twice with the same config and makes sure the same results are shown?